components/private_channel/rust/ffi/BUILD.gn

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//brave/build/cargo.gni")
+import("//brave/build/rust/config.gni")
+
+config("external_config") {
+  if (is_mac) {
+    defines = [ "PRIVATE_CHANNEL_SHARED_LIBRARY" ]
```

Comment:
this needs to be used in the ffi code to export the symbols for the shared lib just like SPEEDREADER_SHARED_LIBRARY, etc..

Reply:
As in https://github.com/brave/brave-core/blob/ca4aab37616f74a740244db64f7901cd7b8a4a05/components/speedreader/rust/ffi/speedreader.h#L14 ? I don't think I understand what is happening here. How should this be used in `private_channel/rust/ffi/src/private_channel.h`?